swiftui/ChartboostMediationDemo/Ads/Banner/BannerAdView.swift

```diff
@@ -59,13 +59,11 @@ struct BannerAdView: View {
                 .buttonStyle(.borderedProminent)
                 .padding(.horizontal, 32)
 
-                if controller.shouldShow, let bannerAd = controller.bannerAd {
+                if controller.shouldShow, let bannerAd = controller.bannerAd, let size = bannerAd.size?.size {
                     Spacer()
                     HStack {
-                        Spacer()
                         Banner(source: bannerAd)
-                            .frame(width: 320, height: 50)
-                        Spacer()
+                            .frame(width: size.width, height: size.height)
```

Comment:
There is a `sizeThatFits` method on `UIViewRepresentable`, and using that allows us to give `Banner` the same intrinsic size as `ChartboostMediationBannerView`, however it's only available on iOS 16+, so manually setting the size of the `Banner` will have to do for now.